.github/workflows/code_size.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build
       run: source tools/ci.sh && ci_code_size_build
     - name: Compute code size difference
-      run: tools/metrics.py diff ~/size0 ~/size1 | tee diff
+      run: source tools/ci.sh && ci_code_size_report
```

Comment:
should the CI scripts be changed to use the `tools/ci.sh code_size_report` style instead (as a separate PR)?